src/mapObs.js

```diff
@@ -59,22 +56,28 @@ const createComponentFromMappers = (mappers, BaseComponent) => {
     }
 
     shouldComponentUpdate(nextProps, nextState) {
-      return this.state.props !== nextState.props;
+      return this.state.childProps !== nextState.childProps;
     }
 
     render() {
-      return factory(this.state.props);
+      if (!this.state.childProps) {
```

Comment:
Yes. My bad. What I really wanted to do was:

1. Not initialising `this.sate`
2. Returning `this.state && this.state.childProps !== nextState.childProps` from `shouldComponentUpdate`.